### PR TITLE
Added trailing slashes to fix redirect

### LINKED ISSUE #303: Relative path error with page links when in development
## Description

Relative path error when in development. 
My branch is up to date with `creativecommons.github.io-source master`. There isn't such an issue on the live site, hence the mention of _development_.

## Reproduction

1. Go to the index page
2. Try clicking `All our projects` in the `Integrate with CC Tools` card.
3. Notice the URL on your browser if a 404

## Expectation

Relative path should work correctly as intended.

## Screenshots

Screenshot from 2020-06-16 14-56-42
Screenshot from 2020-06-16 14-56-46

## Environment

 - Device: laptop
 - OS: Ubuntu 18.04 LTS
 - Browser: Chrome
 - Version: 83.0.4103.61 (Official Build) (64-bit)
 - Other info: 1366x768 (16:9)

## Resolution

- [ ] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by ritesh-pandey**
## Fixes

Fixes #303 by @sp35 

## Description

Absence of trailing slash causes a redirect by server. This PR fixes that by adding trailing slashes. 

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by kgodey** [comment]
@ritesh-pandey I think you should also take a look at the links without trailing slashes in the Markdown formatted links in the content, looks like you're only focusing on the HTML.

I can merge this PR if you'd prefer to do that separately or leave it to someone else.

**Comment by ritesh-pandey** [comment]
I used following expressing to find markdown formatted links.
```
Content in a square bracket followed by parenthesis and a forward slash. This denotes a relative URL.

grep -inr "\[[- a-zA-Z0-9]*\](/[-/ a-zA-Z0-9]*"
```

After that I used a `sed` expression to find and inject trailing slashes.
```
For every regular file in content directory, execute sed expression.
Look for a pattern - 
1. closing square bracket
2. followed by opening parenthesis
3. followed by a slash
4. followed by characters which form a link
5. ends with a alphanumeric character and closing parenthesis

find content/ -type f -print0 | xargs -0 sed -i -r -e 's/\]\(\/([/a-zA-Z0-9-]+)([a-zA-Z0-9])\)/\]\(\/\1\2\/\)/g'
```

File: content/archives/old-tech-blog/entries/ccsearch-testing-progress/contents.lr [hunk 1]
```diff
@@ -9,8 +9,8 @@ body:
 
 Two months ago we set out to develop a brand new interface for [search.creativecommons.org](http://search.creativecommons.org). During that time, two UIs have been in testing alongside the current search. All visitors have had the option to switch their experience over to one of the beta versions.
 
-  1. [A drop-down list](/demos/search/search1) of supported search engines
```
Comment (kgodey) [inline content/archives/old-tech-blog/entries/ccsearch-testing-progress/contents.lr:12]: please undo the changes to this file since this data was imported from an old blog and the content shouldn't be updated

**Comment by kgodey** [comment]
thanks @ritesh-pandey! I've marked this a "Great Contribution" :)
